src/ui/settings/ViewNotifications.qml

```diff
@@ -125,6 +125,21 @@ Item {
                     VPNSettings.connectionChangeNotification = !VPNSettings.connectionChangeNotification
                 }
             }
+
+            VPNCheckBoxRow {
```

Comment:
I’m not sure about the position of the notification option — I’ll have to check with UX.